browser/ui/views/frame/split_view/brave_multi_contents_view.h

```diff
@@ -41,6 +40,8 @@ class BraveMultiContentsView : public MultiContentsView,
   void UpdateSecondaryLocationBar();
   void UpdateCornerRadius();
   void UseContentsContainerViewForWebPanel();
+  void ShowWebPanel(bool show);
```

Comment:
optional: SetVisible(bool visible) could be better

Reply:
Oh yes. That's better

Reply:
Thanks. done.